Makefile

```diff
@@ -136,6 +136,7 @@ clean: FORCE
 	rm -f diff-cover.html
 	rm -Rf build dist
 	rm -rf __pycache__/ .eggs/ khmer.egg-info/
+	-rm *.gcov
```

Comment:
Is there a better way to pattern match these files that has less potential for accidentally deleting stuff that it shouldn't at some point in the future?